packages/databricks-vscode/resources/python/00-databricks-init.py

```diff
@@ -0,0 +1,8 @@
+from databricks.sdk.runtime import dbutils
+from databricks.connect import DatabricksSession
+from pyspark.sql import SparkSession, functions as udf
+
+# "display", "displayHTML", "dbutils", "table", "sql", "udf", "getArgument", "sc", "sqlContext", "spark"
```

Comment:
are these still missing?

Reply:
"displayHTML", "table", "sc", "sqlContext" are still missing. Updated the comment.